collections/cams_glc_2017.yaml

```diff
@@ -1,5 +1,4 @@
 Name: CAMS_GLC_2017
-OpenEOPID: CAMS_GLC
```

Comment:
Why did you remove this? It's needed to generate the ID for https://collections.eurodatacube.com/stac/index.json. We should probably update the contributing.md file with that info.

Reply:
Sorry, I copied the file I edited several months ago. I put it back.